clang/docs/ReleaseNotes.rst

```diff
@@ -217,6 +217,8 @@ Non-comprehensive list of changes in this release
   (e.g., ``uint16x8_t``), this returns the constant number of elements at compile-time.
   For scalable vectors, e.g., SVE or RISC-V V, the number of elements is not known at compile-time and is
   determined at runtime.
+* ``__builtin_classify_type()`` now classifies ``_BitInt`` values as the return value ``18``,
+  to match GCC 14's behavior.
```

Comment:
We should mention that this fixes https://github.com/llvm/llvm-project/issues/71911

Reply:
I didn't add that because the issue also  mentions some extensions to `__builtin_classify_type`. We should probably split that out into a separate issue where we also document the exact extensions, maybe with a link to the gcc commit and test cases.

Reply:
Okay, that's reasonable.